src/symbols.c

```diff
@@ -0,0 +1,80 @@
+#include <string.h>
+
+/*
+ * This file is supposed to be autogenerated in a future.
+ *
+ * Now it contains several internal symbols we want to offer
+ * outside of the module API: no guarantees are provided
+ * regarding them.
+ *
+ * fiber_channel_*() and ipc_value_*() symbols are used in
+ * the Rust module [1], because the symbols were exported in
+ * Tarantool 2.8.
+ *
+ * [1]: https://github.com/picodata/tarantool-module
+ */
+
+extern void
+fiber_channel_close(void);
+extern void
+fiber_channel_create(void);
+extern void
+fiber_channel_delete(void);
+extern void
+fiber_channel_destroy(void);
+extern void
+fiber_channel_get_msg_timeout(void);
+extern void
+fiber_channel_get_timeout(void);
+extern void
+fiber_channel_has_readers(void);
+extern void
+fiber_channel_has_writers(void);
+extern void
+fiber_channel_new(void);
+extern void
+fiber_channel_put_msg_timeout(void);
+extern void
+fiber_channel_put_timeout(void);
+extern void
+ipc_value_delete(void);
+extern void
+ipc_value_new(void);
+
+/** Symbol definition. */
+struct symbol_def {
+	/** Name of the symbol. */
+	const char *name;
+	/** Address of the symbol. */
+	void *addr;
+};
+
+static struct symbol_def symbols[] = {
+	{"fiber_channel_close", (void *)fiber_channel_close},
```

Comment:
Nit: `(void *)` cast is redundant.

Reply:
I was not sure about that in case of a function pointer -> data pointer cast. But GCC 11 and clang 13 give no warnings, so it is likely okay. Changed.